src/test/java/fi/fmi/avi/converter/tac/TACTestConfiguration.java

```diff
@@ -75,6 +76,10 @@
     @Autowired
     private AviMessageSpecificConverter<SpaceWeatherBulletin, String> swxBulletinTACSerializer;
 
+    @Autowired
+    @Qualifier("GenericAviationWeatherMessageTACParser")
+    private AviMessageSpecificConverter<String, GenericAviationWeatherMessage> TACGenericAviationWeatherMessageParser;
```

Comment:
To retain consistent naming I would rename this to `genericAviationWeatherMessageTACParser`.

Reply:
In addition, I think the qualifier is redundant, and the bean name should start with lower case letter. I can change this upon merge.